GENIVI/meta-genivi-dev
# Aktualizr

Hi,

The following changes are provided:
* Align with the latest changes from the master branch
* Update Aktualizr to a newer version with cmake file that allows skipping the build of the tests in the Yocto/OE recipe
* Remove gmock Yocto/OE as it is no longer needed by Aktualizr
* Add systemd service for Aktualizr

Best regards,
Leon
